newsletter/admin.py

```diff
@@ -0,0 +1,26 @@
+from django.conf import settings
+from django.contrib import admin
+
+from . import models
+
+
+@admin.register(models.Subscription)
+class SubscriptionAdmin(admin.ModelAdmin):
+    """
+    Admin for subscription model
+    """
+
+    list_display = [
+        "email",
+        "created_at",
+    ] or settings.NEWSLETTER_ADMIN_LIST_DISPLAY
+
+    readonly_fields = ["first_name", "last_name", "email"] or settings.NEWSLETTER_ADMIN_READONLY_FIELDS
+
+    fieldsets = [
+        (None, {"fields": ["first_name", "last_name", "email"]}),
+        (
+            "Meta Data",
+            {"classes": ("collapse",), "fields": ["created_at", "updated_at"]},
+        ),
+    ] or settings.NEWSLETTER_ADMIN_FIELDSETS
```

Comment:
I believe you'd have to do something like this if you want to use defaults that can be overriden by settings. As it stands, the settings variables won't replace the defaults you've specified even if they exist

```suggestion
    list_display = getattr(
        settings,
        "NEWSLETTER_ADMIN_LIST_DISPLAY", [
            "email",
            "created_at",
         ]
    )

    readonly_fields = getattr(
        settings,
        "NEWSLETTER_ADMIN_READONLY_FIELDS", [
            "first_name",
            "last_name",
            "email",
         ]
    )

    fieldsets = getattr(
        settings,
        "NEWSLETTER_ADMIN_FIELDSETS",
        [
            (None, {"fields": ["first_name", "last_name", "email"]}),
            (
                "Meta Data",
                {"classes": ("collapse",), "fields": ["created_at", "updated_at"]},
            ),
        ]
    )
```

Reply:
^ This'll try to get `settings.NEWSLETTER_ADMIN_FIELDSETS`, and if the attribute doesn't exist uses the specified defaults.

Reply:
Yeah you're right. Good spot